Makefile

```diff
@@ -0,0 +1,5 @@
+start-deps:
+	chmod +x ./docker/build.sh && ./docker/build.sh
+
+start:
+	docker run -it -e ${ARGS} sim-ln:tag
```

Comment:
One thing that I'm missing here is the ability to include a volume that contains the certs/macaroons etc that we'll need to read to talk to the node. 

If our config provides local filesystem paths then sim-cli tries to read them from within the docker file it won't find them.  I think that a simple approach could be to require that the user mounts a single volume that contains the macaroons etc and the config.json and then we look for everything in there. 

Pretty brutal to be dealing with these IPs/paths in the docker context.